src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
```

Comment:
1. The code defines a `GreetingController` class with two `@GetMapping` methods - one for handling the "/greeting" endpoint to return a greeting message and another for handling the "/error" endpoint that deliberately throws a `RuntimeException`.